build.gradle

```diff
@@ -230,7 +230,6 @@ allprojects {
                     dependsOn(':bookkeeper-common:jar')
                     dependsOn(':bookkeeper-common-allocator:jar')
                 }
-                archiveClassifier = "bin"
```

Comment:
the "bin" classifier is needed for the bin tarball
it's the "src" tarball that doesn't need it